src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -3966,9 +3966,9 @@ public ValueTuple(T1 item1, T2 item2)
             Assert.True(mItem1.GetAttributes().IsEmpty);
             Assert.Equal("error CS8128: Member 'Item1' was not found on type 'ValueTuple<T1, T2>' from assembly 'comp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.",
                          mItem1.GetUseSiteDiagnostic().ToString());
-            Assert.False(mItem1.Locations.IsDefaultOrEmpty);
-            Assert.Equal("string", mItem1.DeclaringSyntaxReferences.Single().GetSyntax().ToString());
-            Assert.False(mItem1.IsImplicitlyDeclared);
+            Assert.True(mItem1.Locations.IsDefaultOrEmpty);
+            Assert.True(mItem1.DeclaringSyntaxReferences.IsDefaultOrEmpty);
```

Comment:
Can we be more specific here? Either `IsDefault` or `IsEmpty`? Same question for `Locations` and `DeclaringSyntaxReferences` elsewhere.